.packit.yaml

```diff
@@ -0,0 +1,38 @@
+# https://packit.dev/docs/configuration/
+
+specfile_path: koji-containerbuild.spec
+
+# to be copied to dist-git during an update
+synced_files:
+    - koji-containerbuild.spec
+    - .packit.yaml
+
+# name in upstream package repository/registry (e.g. in PyPI)
+upstream_package_name: koji-containerbuild
+# name of downstream (Fedora) RPM package
+downstream_package_name: koji-containerbuild
+
+# for packit service (Github app)
+jobs:
+- job: sync_from_downstream
+  trigger: commit
+- job: propose_downstream
+  trigger: release
+  metadata:
+    dist-git-branch: master
+- job: propose_downstream
+  trigger: release
+  metadata:
+    dist-git-branch: f31
```

Comment:
Do we want to keep pushing changes to these other branches?
It would be hard to keep packit yaml fiels up to date for all of our repos in the future for the whole fedora release cycle. e.g., f32 gets branched next week.

Maybe we could update rawhide only and have the updates pulled in the previous branches as the maintainer wishes?
Otherwise it would make their life harder at some point (keeping track of what got updated).

It would be nice to have packit understand rawhide - 1 and rawhide -2 though! :)

Reply:
It makes sense to do it only for rawhide. when we integrated packit to all OSBS project we can sync with fedora maintainer how to proceed further.

BTW: can you open RFE to packit? :)